web/src/views/SubmissionPortal/Components/LoginPrompt.vue

```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api';
+
+export default defineComponent({
+  props: {},
```

Comment:
remove

Reply:
Done in commit 794d5b95eef58a33da7c4cbe32cc04e40c6c2778.